src/components/Dashboard.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div> 
+    <table  align="center">
+      <tr v-for="(user, key) in doneCurrent" :key="key">
+        <td class="name">{{user.name}}さんようこそ！！</td>
+        <td>残高:{{user.money}}円</td>
+        <td><button @click="logOut">ログアウト</button></td>
+      </tr>
+    </table>
+    <h1>ユーザ一覧</h1>
+    <div>
+      <table align="center">
+        <tr>
+          <th class="name">ユーザ名</th>
+          <th></th>
+          <th></th>
+        </tr>
+        <tr v-for="(user, key) in doneTodosCount" :key="key">
+          <td class="name">{{user.name}}</td>
+          <td><button @click="openModal(user.id)">walletを見る</button></td>
+          <td><button @click="openSend(user.id)">送る</button></td>           
+        </tr>
+      </table>
+
+      <MyModal @close="closeModal" v-if="modal">
+        <div v-for="(user, key) in  money" :key="key">
+          <p>{{user.name}}の残高</p>
+          <p>{{user.money}}円</p>
+        </div>
+      </MyModal>
+
+      <MyModal @close="closeModal" v-if="throw_money">
+        <div v-for="(user, key) in doneCurrent" :key="key">
+          <div>あなたの残高:{{user.money}}</div>
+        </div>
+          <p>送る金額</p>
+        <div><input v-model="gift"></div>
+        <template slot="footer">
+          <div v-for="(user, key) in money" :key="key">
+            <button @click="doSend(user)">送信</button>
+          </div>
+        </template>
+      </MyModal>
+    </div>
+  </div>
+
+</template>
+
+<script>
+import firebase from 'firebase'
+import MyModal from './MyModal.vue'
+export default {
+components: {MyModal},
+name: 'users',
+  data() {
+    return {
+      modal: false,
+      throw_money: false,
+      gift: '',
+      money:[]
+    }
+  },
+  methods: {
+    doSend(user){
+      this.$store.dispatch('doCurrent',{gift:this.gift,id:user})
```

Comment:
`,`の後ろ`{}`の内側にはスペースを開けるようにしましょう。
```suggestion
      this.$store.dispatch('doCurrent', { gift:this.gift, id:user })
```

Reply:
スペースいれました。